File: src/main/java/net/igneo/icv/entity/chestplate/meteorSummoner/MeteorSummonerRenderer.java
package net.igneo.icv.entity.chestplate.meteorSummoner;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class MeteorSummonerRenderer extends GeoEntityRenderer<MeteorSummonerEntity> {
  public MeteorSummonerRenderer(EntityRendererProvider.Context renderManager) {
    super(renderManager, new MeteorSummonerModel());
  }
}

File: src/main/java/net/igneo/icv/entity/chestplate/meteorSummoner/MeteorSummonerRenderer.kt
package net.igneo.icv.entity.chestplate.meteorSummoner

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class MeteorSummonerRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<MeteorSummonerEntity?>(renderManager, MeteorSummonerModel())
